development/ideas/tensorflow-data-backend.rst

```diff
@@ -23,6 +23,9 @@ efficiently. We hope that by using TensorFlow we might enable QuTiP to scale
 to bigger quantum systems (e.g. more qubits) and decrease the time taken to
 simulate them.
 
+There is particular interest in the trying the new backend with the
+BoFiN HEOM (Hierarchical Euqations of Motion) solver with the new backend [2]_.
```

Comment:
Euqations = Equations..
Backend repeated twice...

Reply:
Fixed! Thank you for catching those. Doh.